src/prmslib/physics/c_transp_tindex.f90

```diff
@@ -26,7 +26,8 @@ MODULE PRMS_TRANSP_TINDEX
       !! Month to begin summing maximum air temperature for each HRU; when sum is greater than or equal to transp_tmax, transpiration begins
     integer(i32), pointer, private :: transp_end(:)
       !! Month to stop transpiration computations; transpiration is computed thru end of previous month
-    real(r32), pointer, private :: transp_tmax(:)
+    !real(r32), pointer, private :: transp_tmax(:)
+    real(r32), pointer :: transp_tmax(:)
```

Comment:
I'm not a fan of leaving commented-out code hanging around; it makes it harder to read the source. I leave it up to @rmcd-mscb and @pnorton-usgs if you want this in PRMS.

Reply:
In this case, yes, that can just be removed. I've removed in the main dev branch. There is commented code in the dev branch that serves a purpose (e.g. stuff that will be uncommented when other modules are converted), but I agree in principal it makes the code harder to read. I periodically will go through the main dev code and remove commented code that is truly no longer needed.

Reply:
Thanks @mdpiper and @pnorton-usgs!  I agree.  I've been leaving comments in prms6 codebase when I make changes, mostly to document those changes.  In developing the bmi's in general we've had to make very few changes to prms6 which is pretty cool!  I guess the documentation is implied of when submitting a pull request so unless otherwise requested I'll make necessary changes without commenting out changed code.  Thanks for the insight!